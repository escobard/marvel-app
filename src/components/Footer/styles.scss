@import "../../styles/variables";
@import "../../styles/mixins";

footer {
  bottom: 0;
  color: $white;
  background-color: $primary;
  transition: all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;
  box-sizing: border-box;
  font-family: $font-family-default;
  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.12), 0 1px 4px rgba(0, 0, 0, 0.12);
  border-radius: 0px;
  z-index: 1100;
  width: 100%;
  display: flex;
  padding: $space-lrg;
  text-align: center;
  span {
    width: 100%;
  }
}
