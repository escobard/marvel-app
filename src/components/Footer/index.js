import React, { Component } from "react";

import styles from "./styles.scss"

export default class Footer extends Component {
	render() {
		return (
			<footer>
				<span>
					{" "}
					&copy; Copyright 2018 -{" "}
					<a href="https://github.com/escobard">escobard@github.com</a>
				</span>
			</footer>
		);
	}
}
