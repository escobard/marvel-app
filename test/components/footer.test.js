import React, { Component } from 'react'
import Footer from '../../src/components/Footer';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'

describe('Footer snapshot', () => {
	const initialState = {output:100}
	const mockStore = configureStore()
	let store,container

    beforeEach(()=>{
        store = mockStore(initialState)
        container = shallow(<Footer /> )  
    })

    it('+++ render the Footer component', () => {
       expect(toJson(container)).toMatchSnapshot()
    });

});