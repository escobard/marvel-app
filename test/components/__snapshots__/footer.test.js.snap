// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot +++ render the Footer component 1`] = `
<footer>
  <span>
     
    Â© Copyright 2018 -
     
    <a
      href="https://github.com/escobard"
    >
      escobard@github.com
    </a>
  </span>
</footer>
`;
